### PR TITLE
Doc/overall review

### PR DESCRIPTION
**Pull request by PipKat**
@Revathyvenugopal162 I finally found time to review the doc and cheat sheets. The doc had cheat sheet as one word instead of two. I also reordered the cheat sheets alphabetically on the doc landing page so that finding one might be easier, especially as the number grows. I'd especially appreciate your careful review of the Linux and Docker content in the README files. I also am assuming that the artifacts created when all checks run successfully will allow me to view the generated PDFs one final time before merging this PR. Please review and provide feedback when you can. Thanks!

### CODE REVIEW

**Comment by PipKat** [comment]
@vnamdeo I made some edits to your new PyDPF-Post cheat sheet in this PR so that it is consistent with the others. I also alphabetized the order in which cheat sheets are listed on the landing page for the PyAnsys Cheat Sheets doc. Note that I will be creating a PR in PyDPF-Post shortly to add links to these new sheets. First, however, I need to know if your new cheat sheet is approved for release. Would you mind confirming that it is? Thanks.

File: cheat_sheets/mapdl_cheat_sheet/pymapdl_cheat_sheet.tex
Comment (Revathyvenugopal162) [inline cheat_sheets/mapdl_cheat_sheet/pymapdl_cheat_sheet.tex:145]:
```suggestion
The \textt{post_processing} class is used for plotting and saving results to NumPy arrays.
```

File: cheat_sheets/pydpf-post_cheat_sheet/pydpf-post_script.py
Comment (vnamdeo) [inline cheat_sheets/pydpf-post_cheat_sheet/pydpf-post_script.py:50]: keep "norm" (not normal)

File: cheat_sheets/pydpf-post_cheat_sheet/pydpf-post_cheat_sheet.tex
Comment (vnamdeo) [inline cheat_sheets/pydpf-post_cheat_sheet/pydpf-post_cheat_sheet.tex:109]: Keep "norm" (not normal)

File: cheat_sheets/pydpf-post_cheat_sheet/pydpf-post_cheat_sheet.tex
Comment (vnamdeo) [inline cheat_sheets/pydpf-post_cheat_sheet/pydpf-post_cheat_sheet.tex:4]: Overall looks good to me except few mentioned below. Thanks for reviewing it.

**Review by vnamdeo — COMMENTED** [review]
Overall looks good to me. Few comments are provided. Thanks !

**Comment by Revathyvenugopal162** [comment]
@PipKat Thank you so much for the review, i can see the pymechanical and pyfluent cheat sheets are misaligned in artifacts. and a minor change in pymapdl cheat sheet Other than that, looks good to me , thanks again for the hardwork.

**Comment by PipKat** [comment]
@Revathyvenugopal162 Fixing alignment sounds like it probably requires your expertise. Learning how to work in LaTeX files isn't the easiest task! Please make those fixes. Also, I'd like to look at the new cheat sheet for PyMotorCAD to implement the same changes as in the others. Should I do this in https://github.com/ansys/pyansys-cheat-sheet/pull/60 after Esther's review?

**Comment by Revathyvenugopal162** [comment]
> @Revathyvenugopal162 Fixing alignment sounds like it probably requires your expertise. Learning how to work in LaTeX files isn't the easiest task! Please make those fixes for me and then also accept and merge this PR! On another note, I'd like to look at the new cheat sheet for PyMotorCAD to implement the same changes as I've made here in this PR. Should I do this in #60 after Esther's review? Or would you prefer I create a new PR after that one is accepted and merged? Lastly, are the PyPDF-Post and PyMotorCAD cheat sheets ready to release to the public? I ask because I'd want to add links to these cheat sheets in their respective documentation.
1. Thanks for the review @PipKat , i resolved all alignment issues
2. It is up to, if you want to add the review in #60, let me know, i can wait until you finish the review and merge it, but if you prefer open another PR, that will also work, pymotorcad will be available public only after your review and merge the PR
3. Pydpf-post will be available after we merge this PR.

**Review by Revathyvenugopal162 — APPROVED** [review]

**Pull request merged**
